axiom/nr_errors_private.h

```diff
@@ -25,6 +25,11 @@ struct _nr_error_t {
   int priority;          /* Error priority - lowest to highest */
   char* message;         /* Error message */
   char* klass;           /* Error class */
+  char* error_file;      /* Error file */
+  int error_line;        /* Error line */
+  char* error_context;   /* Error context */
+  int error_no;          /* Error number */
+  int option;            /* Error option */
```

Comment:
As far as I can tell, these are specifically user-provided through the API. It would be useful to specify that in the name of the variables, or at least their comment.

Reply:
Good point, I have updated the comments: [9f3e7e6](https://github.com/newrelic/newrelic-php-agent/pull/942/commits/9f3e7e652eeeaef0e19c45a1aacbd5ba6891d161)